Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -314,3 +314,9 @@ trait-description-Spinarette =
 trait-name-AddictionNicotine = Nicotine Addiction
 trait-description-AddictionNicotine =
     You have an addiction to Nicotine, and will require frequent smoke breaks to keep your mood in check.
+
+trait-name-ValyrianStandard = Valyrian Standard
+trait-description-ValyrianStandard =
+    A language descended from eastern european languages of old earth - Valyrian Standard is the commonly spoken tongue of Harpies brought up on their homeworld of Valyrian 4b
+    It is rarely spoken outside of the worlds of its native speakers, and has in modern times been supplanted by the 'Conlangs of the Sol Alliance.
+    Its speakers are those who wish to uphold the traditions and beliefs of ancient peoples from before the colonial era.
```

Comment:
```suggestion
    Its speakers are those who wish to uphold the traditions and beliefs of ancient peoples from before the colonial era.

```

Reply:
How do you keep doing this?

Reply:
I have no idea? When I write the PR in VSCode, it includes the Newline.

Reply:
You have EditorConfig right?